lib/Parser/Scan.cpp

```diff
@@ -1778,7 +1778,20 @@ tokens Scanner<EncodingPolicy>::ScanCore(bool identifyKwds)
             {
                 p++;
                 token = tkCoalesce;
-                break;
+            }
+            if (m_scriptContext->GetConfig()->IsEsLogicalAssignmentOperatorEnabled())
+            {
+                Assert(token == tkQMark || token == tkCoalesce);
+                if (token == tkCoalesce && this->PeekFirst(p, last) == '=')
+                {
+                    p++;
+                    token = tkAsgLogCoalesce;
+                }
+                else if (this->PeekFirst(p, last) == '?' && this->PeekFirst(p + 1, last) == '=')
+                {
+                    p += 2;
+                    token = tkAsgLogCoalesce;
+                }
```

Comment:
Is this case only possible if NullishCoalescing is disabled? If yes please include a comment about that.